packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -1,11 +1,7 @@
 import {UpdateHeadOpt} from "@lodestar/fork-choice";
-import {NotReorgedReason} from "@lodestar/fork-choice/lib/forkChoice/interface.js";
+import {NotReorgedReason} from "@lodestar/fork-choice";
 import {ProducedBlockSource} from "@lodestar/types";
-import {
-  BlockSelectionResult,
-  BuilderBlockSelectionReason,
```

Comment:
since we switched to biomjs we no longer catch unused imports with linters this is a bit annoying, I noticed so many unused imports everywhere as well